src/interfaces/adapter.php

```diff
@@ -8,26 +8,19 @@
 namespace Alley\WP\WP_Video_Sync\Interfaces;
 
 use DateTimeImmutable;
-use stdClass;
 
 /**
  * Defines an interface that all adapters must implement.
  */
 interface Adapter {
-	/**
-	 * Fetches the date of the last modification to the last batch of videos.
-	 *
-	 * @return ?DateTimeImmutable
-	 */
-	public function get_last_modified_date(): ?DateTimeImmutable;
```

Comment:
We shouldn't remove this from the interface. Being able to keep a cursor and fetch updated videos since the last run is core to the mission of this library. By removing this from the interface, it grants permission to adapters to not keep a cursor and do a full sync of all videos on every run, which we don't want.

Reply:
ah, good point.